ingest/ingest_pipeline.py

```diff
@@ -61,7 +63,7 @@
 
 class IngestPipeline(object):
     # File location for metadata json convention
-    JSON_CONVENTION = 'DoNotTouch/AMC_v0.8.json'
+    JSON_CONVENTION = 'gs://fc-bcc55e6c-bec3-4b2e-9fb2-5e1526ddfcd2/metadata_conventions/AMC_v1.1.1.json'
```

Comment:
We should use AMC_v1.1.3
I've added the appropriate files to the bucket:
gs://fc-bcc55e6c-bec3-4b2e-9fb2-5e1526ddfcd2/metadata_conventions/AMC_v1.1.3/AMC_v1.1.3.json
gs://fc-bcc55e6c-bec3-4b2e-9fb2-5e1526ddfcd2/metadata_conventions/AMC_v1.1.3/valid_array_v1.1.3.tsv
I think we still need a passing full metadata file before we'd update to AMC_v1.1.3?

Reply:
0dba75b